v19.2/learn-cockroachdb-sql-movr.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Learn CockroachDB SQL
+summary: Learn some of the most essential CockroachDB SQL statements on a local cluster.
+toc: true
+build_for: [standard, managed]
+---
+
+This tutorial walks you through some of the most essential CockroachDB SQL statements, using the `movr` dataset. For a complete list of supported SQL statements and related details, see [SQL Statements](sql-statements.html). For more information about the `movr` dataset and the MovR example application, see [MovR: A Global Ride-sharing App](movr.html).
```

Comment:
These 2 finishing sentences would be easier to scan if they were each on their own line at the end of this intro

Reply:
Done.